app/build.gradle

```diff
@@ -25,11 +26,16 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    kotlinOptions {
+        jvmTarget = '1.8'
```

Comment:
I forgot what this was for - I think for safeargs OR for the directions to be created.